packages/cxl-ui/src/components/cxl-details/cxl-details-checkout.js

```diff
@@ -0,0 +1,37 @@
+import { customElement } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/details';
+import { Details } from '@vaadin/details/src/vaadin-details';
```

Comment:
Is this double-pumping?

Reply:
I believe so. I copied the code from: https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-vaadin-accordion.js and worked from there.